
MoveOptionFlag opt_flags
OptionalFlt64 speed_ratio
